backup.config-example

```diff
@@ -23,6 +23,14 @@ GHE_NUM_SNAPSHOTS=10
 #
 #GHE_RESTORE_HOST="github-standby.example.com"
 
+# Path from where backup data is retrieved (for 'ghe-restore'). By default this  
+# is the "data" directory next to this file but can be set to an absolute path
+# elsewhere for retrieving restore data from a separate partition / mount point.
+# If not set, the 'ghe-restore' utility will use the GHE_DATA_DIR location as  
+# the source location of the restore data.
+#
+#GHE_RESTORE_DATA_DIR="<path-to-github-restore-data>"
```

Comment:
In your description, you mentioned that if GHE_RESTORE_DATA_DIR is not set it'll default to GHE_DATA_DIR. Can you please explain to me where does that happen?

Reply:
it's in https://github.com/github/backup-utils/blob/d30d1eb531851f09765fb5bc62c84d4025027b65/share/github-backup-utils/ghe-backup-config#L195